python/ray/serve/scripts.py

```diff
@@ -466,7 +478,7 @@ def run(
             client.deploy_apps(config, _blocking=False)
             cli_logger.success("Submitted deploy config successfully.")
         else:
-            serve.run(app, host=host, port=port)
+            serve.run(app, route_prefix=route_prefix, host=host, port=port)
```

Comment:
I'm now realizing we need to update this to use `serve.start` to set the `host` and `port` -- can you do that separately?